### PR TITLE
Update to latest base generator

### PR DESCRIPTION
**Pull request by raguiar9080**
This has fixes for the allOf at the root level. Models were regenerated.

PS: I don't particularly like that we've broken compatibility with previous versions. Even though the on wire representation is backwards compatible, the way to use the library is now different so it will require some user changes to accommodate. I'm wondering if we should solve the issue differently on the base generator to keep backwards compatibility, I'll attempt a few things.

### CODE REVIEW

File: src/services/EmissionEstimatesService.ts [hunk 1]
```diff
@@ -548,11 +554,16 @@ export abstract class EmissionEstimatesService {
         id: string,
         data: {
             shipment: Shipment
-            legs: Array<{
-                route: ShippingRoute
-                method: ShippingMethod
-                countryCode?: ShippingCountryCode
-            }>
+            legs: Array<
+                | {
```
Comment (rbruggem) [inline src/services/EmissionEstimatesService.ts:558]: `| {` does this work?

**Comment by raguiar9080** [comment]
After some trials, keeping backwards compatibility for this case is showing up to be quite a hard task to do, will probably require a big change on the base generator and how it generates the services. There's lot of edge cases that make things complicated:
- oneOf inside allOf: the properties need to go outside the `data` field since it's a oneOf type, but the way service templates are this is quite a big change
- allOf with queryParams and how it is presented to the user etc. It would break our current agreement of presenting formData followed by queryParams for example. Can be solved but once again a big change considering the current status quo
- I'm sure I'm forgetting some other edge cases here (like `allOf` inside `allOf`).

Since the current solution correctly handles everything (I believe) I'm leaving it as is.

File: src/services/EmissionEstimatesService.ts (hunk 1 above)
Comment (raguiar9080) [reply src/services/EmissionEstimatesService.ts:558]: 🤔 Didn't notice this. The build succeeds so it does result in 'something'. Let me test locally to see how is this handled.

Comment (raguiar9080) [reply src/services/EmissionEstimatesService.ts:558]: @rbruggem This is actually made like this by the prettify step, it's not there before 👀

So to your question, yes this is working:
Screenshot 2023-06-06 at 11 30 06

Comment (rbruggem) [reply src/services/EmissionEstimatesService.ts:558]: okay!

**Review by rbruggem — APPROVED** [review]

**Comment by raguiar9080** [comment]
> This PR changee the generator to the last one _and_ picked up the latest API specification, right?

Yes, maybe should have been more explicit 🙏 The models were regenerated from latest schema

**Comment by jstasiak** [comment]
I wouldn't necessarily combine these in a single PR, just for the reason that it's not obvious which changes are which

**Pull request merged**